src/main/java/org/gelecekbilimde/scienceplatform/post/controller/CategoryController.java

```diff
@@ -32,10 +29,29 @@ SuccessResponse<List<CategoryResponse>> getCategoryList() {
 		return SuccessResponse.success(categoryResponses);
 	}
 
+	@GetMapping("/{id}")
```

Comment:
@aleyna-yildizli `@RequestMapping`'de tanımlamak yerine aşağıdaki gibi tanımlayabiliriz. Bu sayede listeleme endpotine `/categories` yazabiliriz.
```suggestion
	@GetMapping("/category/{id}")
```